statistics/table.go

```diff
@@ -69,8 +74,6 @@ func (t *Table) copy() *Table {
 		Count:          t.Count,
 		Columns:        make(map[int64]*Column),
 		Indices:        make(map[int64]*Index),
-		colName2Idx:    make(map[string]int64),
```

Comment:
why do you remove these two lines? `copy` means `clone`?

Reply:
1. This two maps is deleted now.
2. Yes

Reply:
gotcha.